packages/react/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,112 @@
+import { Accessibility } from '../../lib/accessibility/types'
+import { ComponentEventHandler, WithAsProp, withSafeTypeForAs } from '../../types'
+import * as _ from 'lodash'
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+import {
+  UIComponentProps,
+  ChildrenComponentProps,
+  commonPropTypes,
+  RenderResultConfig,
+  AutoControlledComponent,
+  applyAccessibilityKeyHandlers,
+} from '../../lib'
+import { textAreaBehavior } from '../../lib/accessibility'
+
+export interface TextAreaSlotClassNames {
+  textArea: string
+}
+
+export interface TextAreaProps extends UIComponentProps, ChildrenComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   */
+  accessibility?: Accessibility
+
+  /** The default value of the text area. */
+  defaultValue?: string
+
+  /**
+   * Called on change.
+   *
+   * @param {SyntheticEvent} event - React's original SyntheticEvent.
+   * @param {object} data - All props.
+   */
+  onChange?: ComponentEventHandler<TextAreaProps>
+
+  /** The value of the text area. */
+  value?: string
+
+  /**
+   * The text area becomes read-only.
+   */
+  disabled?: boolean
+}
+
+export interface TextAreaState {
+  value?: TextAreaProps['value']
+}
+
+class TextArea extends AutoControlledComponent<WithAsProp<TextAreaProps>, TextAreaState> {
+  static className = 'ui-textarea'
+
+  static displayName = 'TextArea'
+
+  static slotClassNames: TextAreaSlotClassNames
```

Comment:
Remove this. You do not have any slots, just the `root` which is `TextArea.className`